### PR TITLE
test: add the Bootstrap css to unit tests

### PR DESCRIPTION
**Pull request by fbasso**
This is a proposal. The purpose of this is:

- Check that everything is alright with the bootstrap css applied. By the way, there were `fixture.detectChanges();` to add in some tests, which were failing with Bootstrap,
- Open some cases to unit tests, for instance for positioning where Bootstrap classes can be required,
- Beautifying the unit tests :)

### CODE REVIEW

**Comment by pkozlowski-opensource** [comment]
Why? I can see the value of Bootstrap CSS in e2e tests but not in unit tests... Sound like we just need to move more to e2e tests instead of "Open some cases to unit tests, for instance for positioning where Bootstrap classes can be required,"....

**Comment by fbasso** [comment]
Thanks for your comment, @pkozlowski-opensource. As I said, it's a proposal, just to discuss about it.

Right, I have no real new use case in view at the moment, the main point for me was "Check that everything is alright with the bootstrap css applied", so that ensure that upgrading the Bootstrap version has no impact on the widgets. 

I had to adapt some tests here, meaning that bootstrap style can have an impact on the unit tests.

Another thing is it's visually better, which render things easier to inspect/debug, especially when it's not our own code.

Last point, there is no impact on performance, the tests run the same speed, so why deprive the tests of a better rendering ?

File: src/karma.conf.js [hunk 1]
```diff
@@ -3,38 +3,29 @@
 const reporters = process.env.TRAVIS ? ['dots'] : ['progress'];
 const browsers = process.env.TRAVIS ? ['ChromeHeadlessNoSandbox'] : ['ChromeNoExtensions'];
 
-module.exports = function (config) {
+module.exports = function(config) {
   config.set({
     basePath: '',
+    files: ['../node_modules/bootstrap/dist/css/bootstrap.min.css'],
```
Comment (maxokorokov) [inline src/karma.conf.js]: please leave only this line in changes, don't reformat the file unnecesary

**Comment by maxokorokov** [comment]
Sill code formatting for `karma.conf.js` pending

**Comment by fbasso** [comment]
@maxokorokov Done ! I also add to adapt one unit test, as the dropdown items are now vertical aligned instead of being on the same line.

**Pull request merged**